src/test/mir-opt/unusual_item_types.core.ptr-drop_in_place.Vec_i32_.AddMovesForPackedDrops.before.mir.64bit

```diff
@@ -0,0 +1,46 @@
+// MIR for `drop_in_place` before AddMovesForPackedDrops
```

Comment:
Not sure why these two new files (`src/test/mir-opt/unusual_item_types.core.ptr-drop_in_place.Vec_i32_.AddMovesForPackedDrops.before.mir.32bit` and `src/test/mir-opt/unusual_item_types.core.ptr-drop_in_place.Vec_i32_.AddMovesForPackedDrops.before.mir.64bit`) appeared.
Perhaps it's a rebase artifact.

Reply:
Indeed - fixed in newly added merge.